### PR TITLE
2021-12-14-distilbert_uncased_te

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/gadde5300/2021-12-14-distilbert_uncased_te.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+val result = pipeline.fit(data).transform(data)
+```
+</div>
+
+## Results
+
+```bash
+| Token    	| Embeddings                                                           	|
+|----------	|----------------------------------------------------------------------	|
+| ఆంగ్ల     	| [1.3526772, 0.66792506, 0.80145407, -1.7625582, 1.222954, ...]        	|
```
Comment (josejuanmartinez) [inline docs/_posts/gadde5300/2021-12-14-distilbert_uncased_te.md:93]: Can you fix the space for the last vertical bar `|`?

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can you please fix one of the vertical bars in the benchmarking? Thank you!

File: docs/_posts/gadde5300/2021-12-14-distilbert_uncased_te.md (hunk 1 above)
Comment (murat-gunay) [reply docs/_posts/gadde5300/2021-12-14-distilbert_uncased_te.md]: @gadde5300 
I fixed the issue here.

**Review by josejuanmartinez — APPROVED** [review]
Looks perfect, thank you!

**Pull request merged**